packages/xchain-bitcoin/src/blockstream-api.ts

```diff
@@ -18,6 +18,9 @@ export const broadcastTx = async ({ network, txHex, blockstreamUrl }: BroadcastT
         return `${blockstreamUrl}/api/tx`
       case Network.Testnet:
         return `${blockstreamUrl}/testnet/api/tx`
+      case Network.Stagenet:
+        // stagenet is not configured, default to mainnet value
+        return `${blockstreamUrl}/api/tx`
```

Comment:
Avoid duplication of code by grouping `Network.Mainnet`|`Network.Stagenet` in `switch/case` statements here and at other places. Especially in places with more logic included (e.g. in xchain-ethereum client etc.).

BEFORE:
```ts
switch (network) {
      case Network.Mainnet:
        return ruleA
      case Network.Testnet:
        return ruleB
      case Network.Stagenet:
          return ruleA
```

CHANGE:

```ts
switch (network) {
      case Network.Mainnet:
      case Network.Stagenet:
        return ruleA
      case Network.Testnet:
        return ruleB
```